crates/re_viewer/src/viewer_analytics/event.rs

```diff
@@ -0,0 +1,130 @@
+use crate::AppEnvironment;
```

Comment:
Now that `viewer_analytics` is a mod dir, we should move `viewer_analytics.rs` -> `viewer_analytics/mod.rs`